### PR TITLE
AC-7350 :: Winner Statuses are included on People and Expert Profiles

### PR DESCRIPTION
**Pull request by freshprincekla**
**Changes introduced in [AC-7350](https://masschallenge.atlassian.net/browse/AC-7350)**
 -  Added Winner Status label to the expert profile, entrepreneur profile and startup cards.

**How to test**
- Checkout this branch `AC-7572v2` on both `impact-api`  and `front-end`
- See test instructions on related [PR](https://github.com/masschallenge/front-end/pull/236)

**Note:** Named the branch incorrectly

### CODE REVIEW

**Comment by iankigen** [comment]
Doing QA

File: web/impact/impact/graphql/types/entrepreneur_profile_type.py [hunk 1]
```diff
@@ -49,6 +49,14 @@ def resolve_title(self, info, **kwargs):
 
     def resolve_program_roles(self, info, **kwargs):
         user_roles_of_interest = [UserRole.FINALIST, UserRole.ALUM]
-        startup_roles_of_interest = [StartupRole.ENTRANT]
+        startup_roles_of_interest = [
+            StartupRole.ENTRANT,
+            StartupRole.WINNER,
```
Comment (iankigen) [inline web/impact/impact/graphql/types/entrepreneur_profile_type.py]: why not use `startup_roles_of_interest  += StartupRole.WINNER_STARTUP_ROLES`

File: web/impact/impact/graphql/types/startup_type.py [hunk 2]
```diff
@@ -27,3 +28,8 @@ def resolve_name(self, info, **kwargs):
     def resolve_high_resolution_logo(self, info, **kwargs):
         if self.high_resolution_logo:
             return self.high_resolution_logo.url
+
+    def resolve_program_startup_status(self, info, **kwargs):
+        return self.program_startup_statuses().filter(
+            startupstatus__startup=self).values_list(
```
Comment (iankigen) [inline web/impact/impact/graphql/types/startup_type.py]: you don't need to add a filter:`self.program_startup_statuses().filter(startupstatus__startup=self)` is the same as `self.program_startup_statuses()`

**Review by iankigen — APPROVED** [review]
Code/QA looks good:
- Winner Statuses are included on People and Expert Profiles
- Continuously grant winner status to new team members who are added after an award has been given

**Review by shakirandagire — APPROVED** [review]
Code changes look good to me

**Pull request merged**